sdk/core/README.md

```diff
@@ -124,11 +124,20 @@ const wallet = AffinidiWallet.openWalletByEncryptedSeed(options, encryptedSeed,
 
 ### Create a new wallet
 
+'elem-anchored' did method returns did:elem, and it is anchored with [sidetree](https://identity.foundation/sidetree/spec/) in [ropsten testnet](https://ropsten.etherscan.io/)  
+note: elem-anchored doesn't support external keys due to Sidetree protocol limitations
```

Comment:
Do we want to add this check into `ElemAnchoredDidService.buildDidDocumentForRegister`?

Reply:
OK, just noticed this discussion https://lemmatree.slack.com/archives/C025J7PEG80/p1632996281380700?thread_ts=1632996163.380500&cid=C025J7PEG80

Reply:
According to @DenisPopov15, as it is implemented on affinity-registry side I don't see any reasons, and more, if the sidetree standard will change and will allow more space it means we need to keep this moment in mind for this codebase